numpy/__init__.pyi

```diff
@@ -463,27 +465,24 @@ class float32(floating):
 class float64(floating):
     def __init__(self, __value: Optional[SupportsFloat] = ...) -> None: ...
 
-class complexfloating(inexact): ...  # type: ignore
+class complexfloating(inexact, Generic[_FloatType]):  # type: ignore
+    @property
+    def real(self) -> _FloatType: ...
+    @property
+    def imag(self) -> _FloatType: ...
+    def __abs__(self) -> _FloatType: ...  # type: ignore[override]
```

Comment:
Why is `ignore` needed here but not on the others?

Reply:
`__abs__()` is already defined a super-class while the other two are not (as per our previous `real`/`imag` discussion):

https://github.com/numpy/numpy/blob/32b3f82e4862d85c5aebba2b2b82a931038cd103/numpy/__init__.pyi#L256

Mypy will complain due to incompatibilities between the super- and sub-types' signatures, 
complaints which are silenced by `# type: ignore[override]`.

For more details: https://mypy.readthedocs.io/en/stable/common_issues.html#incompatible-overrides